Translation Dream By Bi-Chan

lquattro opened the pull request:
- Translate Dream Generic EN to FR 26 lines.

**Actualy : Work in Progress** Please not merge now.

bi-chan committed:
Update 16-10-2016

Traduit les phrases de 0 a 26 aux lignes 3761 à 3787.

lquattro commented on translations/stonehearth/fr-FR.json:
```diff
@@ -3758,33 +3758,33 @@
-                     "025": "I dreamed that one of the ancient trees came alive and attacked our town! It was horrible.",
-                     "026": "I dreamed about a time where houses didn't have roofs. It was very cold.",
+                     "000": "En premier, j'ai rêvé que je trouvais un mirroir scintillant sous mon oreiller. Puis, les goblins attaquaient notre caravane.",
+                     "001": "J'ai rêvé que je construisais une échelle jusqu'au ciel! C'est alors que quelqu'un l'arracha et me laissa là haut. C'était effrayant.",
```
Sur cette ligne par exemple (voir les autres) il y a une faute de ponctuation :
En français, on doit mettre des espaces devant des points d'exclamation et d'interrogation.

bi-chan commented:
Allo,

j'ai vu ton commentaire et j'ai fais une petite recherche.


L'office de la langue québécoise :

http://www.oqlf.gouv.qc.ca/francilettre/pdf_franc8/ExtraitCCLQTI_Espacements.pdf


Ils disent qu'il n'y a pas d'espacement avant les point d'exclamation et les point d'interrogation.


De plus, dans mon bac ce site est une de nos référence par excellence.


http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2039

Ils disent que s'il y a un espace elle ne doit pas être grosse ( donc mieux si on en met pas.).


Donc dépendamment d'où on vient il y a peut-être une différence? Par contre, c'est un réflexe de ne pas en mettre puisque je n'en ai jamais mis. Peut-être faudra-t-il vérifié dorénavant pour ces petits détails et ces différences locales.


Bonne journée!

Banque de dépannage linguistique - Espacement avant et ...<http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2039>
bdl.oqlf.gouv.qc.ca
Le tableau de cet article indique si les principaux signes de ponctuation et symboles doivent être précédés ou suivis d'un espacement, et de quel type d ...



Extrait du document disponible dans le site Web de l ...<http://www.oqlf.gouv.qc.ca/francilettre/pdf_franc8/ExtraitCCLQTI_Espacements.pdf>
www.oqlf.gouv.qc.ca
page 1 de 2 Extrait du document disponible dans le site Web de l'Office québécois de la langue française : Caractéristiques culturelles et linguistiques du Québec





________________________________
De : lquattro <notifications@github.com>
Envoyé : 19 octobre 2016 08:38
À : Benjamin124/startermod_locale_fr
Cc : bi-chan; Assign
Objet : Re: [Benjamin124/startermod_locale_fr] Translation Dream By Bi-Chan (#19)


@lquattro commented on this pull request.

________________________________

In translations/stonehearth/fr-FR.json<https://github.com/Benjamin124/startermod_locale_fr/pull/19#pullrequestreview-4860062>:

> -                     "013": "I dreamed that I was at the mercy of a sickness that turned me into a Zombie. The worst part: I could only walk very very slowly.",
-                     "014": "Last night I dreamed I was stranded in a hole all night and no one would build a ladder for me to climb up.",
-                     "015": "I dreamed that all the animals plotted against us and took over the town.",
-                     "016": "I dreamed that poyos rained from the sky.",
-                     "017": "I dreamed that a giant man-bunny threatened me with a spear!",
-                     "018": "I dreamed I found a mighty hammer that only I could lift.",
-                     "019": "I dreamed I was a vegetable and I planted myself so I can grow taller.",
-                     "020": "In my dream last night, I told everyone carrots made them see in the dark. This morning, we ran out of carrots. Weird!",
-                     "021": "I dreamed that I was no longer tethered to this realm, and I drifted from plane to plane.",
-                     "022": "I had a dream that people buried me and made me into a tree.",
-                     "023": "I dreamed the trees could talk.",
-                     "024": "I dreamed I spoke to a rock, but it was stone cold towards me.",
-                     "025": "I dreamed that one of the ancient trees came alive and attacked our town! It was horrible.",
-                     "026": "I dreamed about a time where houses didn't have roofs. It was very cold.",
+                     "000": "En premier, j'ai rêvé que je trouvais un mirroir scintillant sous mon oreiller. Puis, les goblins attaquaient notre caravane.",
+                     "001": "J'ai rêvé que je construisais une échelle jusqu'au ciel! C'est alors que quelqu'un l'arracha et me laissa là haut. C'était effrayant.",


Sur cette ligne il y a une faute de ponctuation :
En français, on doit mettre des espaces devant des points d'exclamation et d'interrogation.

-
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/Benjamin124/startermod_locale_fr/pull/19#pullrequestreview-4860062>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQOFEPrrWJn8l1EMnBqdThLs5GV4dqrEks5q1g9HgaJpZM4Katz2>.

bi-chan committed:
Update 30-10-2016

Traduit phrase generic de 27-49
Lignes 3789-3810
Pas encore fait la correction de mes phrases a 100%

bi-chan committed:
Update 03-01-2017

Terminer ma partie entre la ligne 3737 et 3860.
Continuer dans les lignes plus bas vers 4066 a 4089; de 4199 a 4236.

bi-chan committed:
Update 04-01-2017

Traduit les lignes : 4096 à 4293 inclusivement.

Wiese2007 commented:
perhaps add her as contributor ;)
